db/migrate/20230925123457_create_design_files.rb

```diff
@@ -0,0 +1,10 @@
+class CreateDesignFiles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :design_files do |t|
```

Comment:
Consider adding validations to ensure data integrity. For example, you might want to validate the presence of the 'name' attribute and ensure 'last_modified' is a valid datetime.